### PR TITLE
chore: codegen test skip v8 failed test

### PR DESCRIPTION
**Pull request by underfin**
Because v8 isn't pass some test262 tests, we also need to skip them, you can find it at https://chromium.googlesource.com/v8/v8/+/refs/heads/main/test/test262/test262.status.

### CODE REVIEW

File: tasks/coverage/src/runtime/mod.rs [hunk 1]
```diff
@@ -182,7 +201,7 @@ async fn request_run_code(json: impl serde::Serialize + Send + 'static) -> Resul
     tokio::spawn(async move {
         agent()
             .post("http://localhost:32055/run")
-            .timeout(Duration::from_secs(10))
+            .timeout(Duration::from_secs(50))
```
Comment (Boshen) [inline tasks/coverage/src/runtime/mod.rs:204]: This seems like an unrelated change? Why do we need to timeout for so long?

This goes against the philosophy of this project, we should strive to keep all the waiting times down.

File: tasks/coverage/src/lib.rs [hunk 2]
```diff
@@ -80,6 +81,26 @@ impl AppArgs {
...
+    pub fn run_sync_v8_test262_status(&self) {
+        let res = agent()
+            .get("http://raw.githubusercontent.com/v8/v8/main/test/test262/test262.status")
+            .timeout(Duration::from_secs(10))
+            .call()
+            .expect("Get v8 test262 status failed")
+            .into_string()
+            .expect("Get v8 test262 status failed");
+        let mut content = String::new();
+        regex::Regex::new(r"'(.+)': \[FAIL\]").unwrap().captures_iter(&res).for_each(|caps| {
```
Comment (Boshen) [inline tasks/coverage/src/lib.rs:96]: There are also [SKIP] Markers

File: tasks/coverage/src/lib.rs [hunk 3]
```diff
@@ -80,6 +81,26 @@ impl AppArgs {
...
+            content.push_str(caps.get(1).unwrap().as_str());
+            content.push('\n');
+        });
```
Comment (Boshen) [inline tasks/coverage/src/lib.rs:99]: Can we also sort the lines? So we don't get a big diff if the file ever changes.

File: tasks/coverage/src/runtime/mod.rs (hunk 1 above)
Comment (underfin) [reply tasks/coverage/src/runtime/mod.rs:204]: Here has many timeout at my local environment, it is valid to avoid timeout.

Comment (Boshen) [reply tasks/coverage/src/runtime/mod.rs:204]: We need to fix the timeout if this is the case. cc @Dunqing if you happen to know why it's timing out.

Comment (underfin) [reply tasks/coverage/src/runtime/mod.rs]: Could you also help me update the test snapshot file? Thanks..

**Review by Boshen — APPROVED** [review]

**Comment by Boshen** [comment]
Thank you!

File: tasks/coverage/src/runtime/mod.rs (hunk 1 above)
Comment (Boshen) [reply tasks/coverage/src/runtime/mod.rs]: Try `just submodules`. You may need to delete their directories.
